lib/cacheable/response_cache_handler.rb

```diff
@@ -140,7 +140,7 @@ def serve_from_cache(cache_key_hash, cache_age_tolerance=nil, message = "Cache h
               body = Cacheable.decompress(body)
             end
 
-            render text: body, status: status
+            render plain: body, status: status
```

Comment:
This means we need to change the required version of action pack. It is 3.2 but `render plain:` was only added in Rails 4.1.